acceptance/chaos_test.go

```diff
@@ -113,27 +115,39 @@ func TestChaos(t *testing.T) {
 				return
 			default:
 			}
+			atomic.StoreInt64(&roundCopy, int64(round))
```

Comment:
pretty sure you don't need the `int64()` here since `round` is initialized to an untyped constant